.ci/schedule-daily.groovy

```diff
@@ -37,7 +31,37 @@ pipeline {
   }
 }
 
-def runBuild(Map args = [:]) {
-  def jobName = args.job
-  build(quietPeriod: args.quietPeriod, job: jobName, parameters: [booleanParam(name: 'macosTest', value: true)], wait: false, propagate: false)
+def runBuilds(Map args = [:]) {
+  def branches = []
+  // Expand macros and filter duplicated matches.
+  args.branches.each { branch ->
+    def branchName = getBranchName(branch)
+    if (!branches.contains(branchName)) {
+      branches << branchName
+    }
+  }
+
+  def quietPeriod = 0
+  branches.each { branch ->
+    build(quietPeriod: quietPeriod, job: "Beats/beats/${branch}", parameters: [booleanParam(name: 'macosTest', value: true)], wait: false, propagate: false)
+    // Increate the quiet period for the next iteration
+    quietPeriod += quietPeriodFactor
```

Comment:
I think we are missing `quietPeriodFactor` defined or read from args

Reply:
Good catch, it's `args.quietPeriodFactor`

```suggestion
    quietPeriod += args.quietPeriodFactor
```